src/simtools/applications/derive_photon_electron_spectrum.py

```diff
@@ -82,10 +82,24 @@ def _parse(label):
         required=False,
     )
     config.parser.add_argument(
-        "--threshold_afterpulse_spectrum",
-        help="Threshold for afterpulse ratio calculation",
+        "--afterpulse_amplitude_range",
+        help="Amplitude range in pe for afterpulse calculation",
         type=float,
-        default=4.0,
+        nargs=2,
+        default=[0.0, 42.0],
+        required=False,
+    )
+    config.parser.add_argument(
+        "--fit_afterpulse",
+        help="Fit afterpulse spectrum with an exponential decay function.",
+        action="store_true",
+        required=False,
+    )
+    config.parser.add_argument(
+        "--fit_afterpulse_decay_factor",
```

Comment:
Should this be `--fix_afterpulse_decay_factor`? If so, perhaps a better name would be `afterpulse_decay_factor_fixed_value`? Though I am really not sure so please don't accept it just cause I suggested.

Reply:
It should be 'fix'... I've taken your suggestion.